polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -1425,6 +1452,57 @@ pub mod pallet {
 				weight_limit,
 			)
 		}
+
+		/// Authorize another `aliaser` location to alias into the local `origin` making this call.
+		///
+		/// Usually useful to allow your local account to be aliased into from a remote location
+		/// also under your control (like your account on another chain).
+		///
+		/// WARNING: make sure the caller `origin` (you) trusts the `aliaser` location to act in
+		/// their/your name. Once authorized using this call, the `aliaser` can freely impersonate
+		/// `origin` in XCM programs executed on the local chain.
+		#[pallet::call_index(14)]
+		pub fn add_authorized_alias(
+			origin: OriginFor<T>,
+			aliaser: Box<VersionedLocation>,
+		) -> DispatchResult {
+			let origin: Location = T::ExecuteXcmOrigin::ensure_origin(origin)?;
+			let aliaser: Location = (*aliaser).try_into().map_err(|()| Error::<T>::BadVersion)?;
+			tracing::debug!(target: "xcm::pallet_xcm::add_authorized_alias", ?origin, ?aliaser);
+			ensure!(origin != aliaser, Error::<T>::BadLocation);
+			let v_origin = VersionedLocation::from(origin);
+			let v_aliaser = VersionedLocation::from(aliaser);
+			let mut authorized_aliases = AuthorizedAliasesMap::<T>::get(&v_origin);
+			if !authorized_aliases.contains(&v_aliaser) {
+				authorized_aliases
+					.try_push(v_aliaser)
+					.map_err(|_| Error::<T>::TooManyAuthorizedAliases)?;
+				AuthorizedAliasesMap::<T>::insert(&v_origin, authorized_aliases);
+			}
```

Comment:
```suggestion
			let versioned_origin = VersionedLocation::from(origin);
			let versioned_aliaser = VersionedLocation::from(aliaser);
			let mut authorized_aliases = AuthorizedAliasesMap::<T>::get(&versioned_origin);
			if !authorized_aliases.contains(&versioned_aliaser) {
				authorized_aliases
					.try_push(versioned_aliaser)
					.map_err(|_| Error::<T>::TooManyAuthorizedAliases)?;
				AuthorizedAliasesMap::<T>::insert(&versioned_origin, authorized_aliases);
			}
```

I really dislike abbreviations 😅